src/CMakeLists.txt

```diff
@@ -141,32 +141,6 @@ set(keepassx_SOURCES_MAINEXE
     main.cpp
 )
 
-set(keepassx_FORMS
```

Comment:
CMake finds them automatically with the new cmake_autouic define. Only downside is you may have to clear out the old ui headers that were autogenerated using the previous method.